### PR TITLE
toast component works and all alerts are replaced

### PR DESCRIPTION
**Pull request by florianblaschke**
I still have an error in deployment and can't figure out why. Any suggestions? -> Solved. Never ever again will I change fileNames and let VSCode handle the imports. Oh. My. Gosh.

### CODE REVIEW

File: pages/login/index.js [hunk 1]
```diff
@@ -54,14 +56,16 @@ export default function Profile() {
     });
 
     if (res.status === 418) {
-      return alert("Sry, dieser Name ist leider schon vergeben!");
+      return toast.errorToast("Sry, dieser Name ist leider schon vergeben!");
```
Comment (MerleMak) [inline pages/login/index.js:59]:
```suggestion
      return toast.errorToast("Sorry, dieser Name ist leider schon vergeben!");
```

File: pages/addroast.js [hunk 2]
```diff
@@ -18,14 +20,19 @@ export default function AddRoast() {
       body: JSON.stringify(data),
     });
     if (res.status === 400) {
-      alert("This coffee already exists");
+      toast.errorToast("Diesen Kaffee kennen wir bereits!");
     }
     if (res.status === 418) {
-      alert("You can't submit empty fields and/or just numbers.");
+      toast.errorToast("Sry – leere Felder und/oder nur Zahlen gehen nicht!");
```
Comment (MerleMak) [inline pages/addroast.js:26]:
```suggestion
      toast.errorToast("Sorry – leere Felder und/oder nur Zahlen gehen nicht!");
```

File: components/Modals/Toast/toastBox.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+const ToastContainer = styled.div`
+  display: flex;
+  flex-flow: column-reverse;
+  position: fixed;
+  top: 0px;
+  width: 100%;
+  align-items: center;
+  font-size: 1.2rem;
+  z-index: 9999;
```
Comment (MerleMak) [inline components/Modals/Toast/toastBox.js:27]:
```suggestion
  z-index: 1;
```

**Review by MerleMak — APPROVED** [review]
This looks very very nice! Really good job :)
One thing that i noticed, but that doesn't have anything to do with the toasts: The button for turning a coffee into a favourite: I would be amazing if it was a toggle button, meaning that you can just click it again an "unfavoritze" the coffee right there on the spot instead of needing to go to the profile :) This would include to add another toast message like "this coffee is not a favourite anymore"
But yeah besides that its all really nicely implemented. A lot of toast messages, but i respect that :D Keep it going 🚀

**Comment by florianblaschke** [comment]
> This looks very very nice! Really good job :) One thing that i noticed, but that doesn't have anything to do with the toasts: The button for turning a coffee into a favourite: I would be amazing if it was a toggle button, meaning that you can just click it again an "unfavoritze" the coffee right there on the spot instead of needing to go to the profile :) This would include to add another toast message like "this coffee is not a favourite anymore" But yeah besides that its all really nicely implemented. A lot of toast messages, but i respect that :D Keep it going 🚀

Hi! Thank you! The toggleFavorite will be coming!

**Pull request merged**